### DISCUSSION TITLE
Adding an image in read_index.html is not working

### DISCUSSION THREAD

**Discussion by Martayugui** [bluenviron/mediamtx#3633] [General]
Hi everybody,

I am customizing the web page via read_index.html file. One of the new features I would like to add is to display a image in the middle of the screen when video streams are not coming.
I have run the same test with another html file and it has worked.

I have added the image using some JS code, CSS and HTML:

```
#header img {

position: fixed;
top: 50%;
left: 50%;
transform: translate(-50%, -50%);
width: 20%;
}
```

`<div id="header" class="center"></div>`

         
```
            var img = document.createElement("img");
            img.src = "photo.svg";
            var src = document.getElementById("header");
            src.appendChild(img);

```

What I am getting:
![image](https://github.com/user-attachments/assets/b7d7f89f-ad09-4a7a-9e42-b6259dd5d817)

I have tried different ways to show the image but any of them have worked.
Any idea?

Thank you in advance!

**Comment by aler9** [comment]
Hello, if you want to customize the WebRTC reader, the README suggests "to create and serve a custom web page by starting from the [source code of the WebRTC publish page](https://github.com/bluenviron/mediamtx/blob/main/internal/servers/webrtc/publish_index.html)", not to hack the server.

The server doesn't support hosting resources other than the web page itself.

Conversely, if you take the code of the WebRTC page and host it on your own HTTP server (for instance, nginx), you will be able to host additional resources, like images.

[a comment by an automated account is not shown]
